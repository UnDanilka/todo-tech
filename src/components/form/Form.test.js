import React from 'react';
import Form from './Form';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({adapter: new Adapter()});




 


describe('testing Form', ()=>{
    it('Form have rendered correctly',  ()=>{
        const form = shallow(<Form />);
        expect(form).toMatchSnapshot();
    })
})