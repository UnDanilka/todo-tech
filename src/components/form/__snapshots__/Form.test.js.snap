// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Form Form have rendered correctly 1`] = `ShallowWrapper {}`;
